lib/evmone/eof.cpp

```diff
@@ -364,7 +364,8 @@ std::variant<EOFValidationError, int32_t> validate_max_stack_height(
 
         const auto opcode = static_cast<Opcode>(code[i]);
 
-        auto stack_height_required = instr::traits[opcode].stack_height_required;
+        auto stack_height_required =
```

Comment:
```suggestion
        int stack_height_required =
```
The arithmetic is done with `int` precision at least. This will avoid a lot of casts.

Reply:
in 76ad9260 i did it in `uint8_t` and most of the casts are gone. I also think it should be enough for the `.inputs` case, inputs are 1 byte wide, and don't fill that space up as per current spec. Does this work?